modules/openapi-generator/src/main/resources/typescript-fetch/runtime.mustache

```diff
@@ -3,6 +3,9 @@
 
 export const BASE_PATH = "{{{basePath}}}".replace(/\/+$/, "");
 
+const isBlob = (value: any) => typeof Blob !== 'undefined' && value instanceof Blob;
+const isBuffer = (value: any) => typeof Buffer !== 'undefined' && value instanceof Buffer;
```

Comment:
maybe it is easier to only support Blob, since `Buffer` does not exist in the `dom` typings https://github.com/Microsoft/TypeScript/blob/master/lib/lib.dom.d.ts

Reply:
It should be enough to install `@types/node`, which seems pretty sensible.

Reply:
but I imagine a user wouldn't want to install `@types/node` for a web project, since the node API won't be available in the web...

Reply:
I don't think a user would care much... create-react-app for typescript installs @types/node too.